### PR TITLE
Validates project license header

### PR DESCRIPTION
**Pull request by alecharp**
Because I'm updating the copyright header on files which were not imported by me, please @dheerajodha, let me know here if you agree to the license.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -21,7 +21,6 @@
   ~ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
   ~ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
   ~ SOFTWARE.
-  ~
```
Comment (alecharp) [inline pom.xml:24]: This is just an "empty" line removable, but do you agree to this @dheerajodha, as this file was added by you?

File: src/main/java/io/jenkins/pluginhealth/scoring/probes/DeprecatedPluginProbe.java [hunk 2]
```diff
@@ -1,3 +1,27 @@
+/*
```
Comment (alecharp) [inline src/main/java/io/jenkins/pluginhealth/scoring/probes/DeprecatedPluginProbe.java:1]: This file was originally created by @dheerajodha. Do you agree to this license here?

File: src/main/java/io/jenkins/pluginhealth/scoring/probes/LastCommitDateProbe.java [hunk 3]
```diff
@@ -1,3 +1,27 @@
+/*
```
Comment (alecharp) [inline src/main/java/io/jenkins/pluginhealth/scoring/probes/LastCommitDateProbe.java:1]: This file was originally created by @dheerajodha. Do you agree to this license here?

File: src/main/java/io/jenkins/pluginhealth/scoring/probes/UpForAdoptionProbe.java [hunk 4]
```diff
@@ -1,3 +1,27 @@
+/*
```
Comment (alecharp) [inline src/main/java/io/jenkins/pluginhealth/scoring/probes/UpForAdoptionProbe.java:1]: This file was originally created by @dheerajodha. Do you agree to this license here?

File: src/test/java/io/jenkins/pluginhealth/scoring/service/PluginServiceIT.java [hunk 5]
```diff
@@ -1,3 +1,27 @@
+/*
```
Comment (alecharp) [inline src/test/java/io/jenkins/pluginhealth/scoring/service/PluginServiceIT.java:1]: This file was originally created by @dheerajodha. Do you agree to this license here?

File: pom.xml (hunk 1 above)
Comment (dheerajodha) [reply pom.xml:24]: I agree.

File: src/main/java/io/jenkins/pluginhealth/scoring/probes/DeprecatedPluginProbe.java (hunk 2 above)
Comment (dheerajodha) [reply src/main/java/io/jenkins/pluginhealth/scoring/probes/DeprecatedPluginProbe.java:1]: I agree.

File: src/main/java/io/jenkins/pluginhealth/scoring/probes/LastCommitDateProbe.java (hunk 3 above)
Comment (dheerajodha) [reply src/main/java/io/jenkins/pluginhealth/scoring/probes/LastCommitDateProbe.java:1]: I agree.

File: src/main/java/io/jenkins/pluginhealth/scoring/probes/UpForAdoptionProbe.java (hunk 4 above)
Comment (dheerajodha) [reply src/main/java/io/jenkins/pluginhealth/scoring/probes/UpForAdoptionProbe.java:1]: I agree.

File: src/test/java/io/jenkins/pluginhealth/scoring/probes/LastCommitDateProbeTest.java [hunk 6]
```diff
@@ -1,3 +1,27 @@
+/*
```
[the remark this answers is not in the document]
Comment (dheerajodha) [reply src/test/java/io/jenkins/pluginhealth/scoring/probes/LastCommitDateProbeTest.java:1]: I agree.

File: src/test/java/io/jenkins/pluginhealth/scoring/service/PluginServiceIT.java (hunk 5 above)
Comment (dheerajodha) [reply src/test/java/io/jenkins/pluginhealth/scoring/service/PluginServiceIT.java:1]: I agree.

**Review by dheerajodha — APPROVED** [review]
Thanks for the mention. I approve this PR for adding license headers to the files originally created by me.

**Pull request merged**